packages/webpack-utils/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as fabricAsyncLoader from './fabricAsyncLoader';
```

Comment:
Added an index file to this package to better match convention (also it was already referenced in package.json but didn't exist...)